psi4/src/psi4/dfocc/dfocc.cc

```diff
@@ -51,6 +51,8 @@ void DFOCC::common_init() {
     print_ = options_.get_int("PRINT");
     if (print_ > 0) options_.print();
 
+    module_ = "dfocc";
```

Comment:
Just to double check:
- Inherits from Wavefunciton uses `module_ = ...`
- Does not inherit from Wavefunction uses `->set_module()`?

Reply:
or, `.set_module()` py-side, yes. I used whatever was closest to the class. No strong opinions, though.